client_end_user/shared/src/commonMain/kotlin/presentation/SearchTaxi/SearchTaxiScreen.kt

```diff
@@ -0,0 +1,26 @@
+package presentation.SearchTaxi
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import cafe.adriel.voyager.navigator.Navigator
+import presentation.base.BaseScreen
+
+class SearchTaxiScreen : BaseScreen<SearchTaxiScreenModel,SearchTaxiUIState, SearchTaxiUIEffect, SearchTaxiInteractionListener>() {
+
+
+    override fun onEffect(effect: SearchTaxiUIEffect, navigator: Navigator) {
+        when(effect){
+            SearchTaxiUIEffect.onGoBack -> navigator.pop()
+        }
+    }
+
+    @Composable
+    override fun Content() {
+        initScreen(getScreenModel())
+    }
+
+    @Composable
+    override fun onRender(state: SearchTaxiUIState, listener: SearchTaxiInteractionListener) {
+        Box(){}
```

Comment:
add todo comment
don't leave empty composable

Reply:
@Saif-Alhaider  the branch it's draft and  stil not working on it 😶